ansible/roles/ovos_installer/templates/virtualenv/extra-skills-requirements.txt.j2

```diff
@@ -0,0 +1,12 @@
+# No PyPi release
+git+https://github.com/OpenVoiceOS/skill-ovos-icanhazdadjokes.git
+git+https://github.com/OpenVoiceOS/skill-ovos-fallback-chatgpt.git
+git+https://github.com/OpenVoiceOS/skill-ovos-parrot.git
+git+https://github.com/OpenVoiceOS/skill-ovos-randomness.git
+git+https://github.com/OpenVoiceOS/skill-ovos-tunein.git
+git+https://github.com/OpenVoiceOS/skill-ovos-wolfie.git
```

Comment:
suggest adding the new speed test skill also https://github.com/OpenVoiceOS/ovos-skill-speedtest

Reply:
SpeedTest has not been tested enough IMO to be part of this list yet _(no Docker image as well)_.